### ISSUE TITLE
Include ideal python version in README

### ISSUE THREAD

**Issue by nramin** [shaduzlabs/cabl#8]
Hi, I had a lot of trouble initially building this on Mac OS X until I realized cmake was using Python 3.6. You may want to put Python 2.7 in the readme so people know what version to use.

**Comment by shaduzlabs** [comment]
Good point! I added some info in the README file on the *develop* branch
